recipe/meta.yaml

```diff
@@ -32,22 +32,25 @@ requirements:
     - python    # [win]
     - cmake >=3.1
     - zlib 1.2*
+    - gcc  # [not win]
```

Comment:
Yes agree about `libfortran` but `gcc` should have the `linux` selector instead? Is that right?